projects/rero/ng-core/src/lib/record/search/aggregation/buckets/buckets.component.ts

```diff
@@ -105,17 +104,7 @@ export class BucketsComponent implements OnInit, OnDestroy, OnChanges {
     const aggregationFilters = this.aggregationsFilters.find(
       (item: AggregationsFilter) => item.key === this.aggregationKey
     );
-    return (aggregationFilters === undefined)
-      ? []
-      : aggregationFilters.values;
-  }
-
-  /**
-   * Get current language
-   * @return Current language
-   */
-  get language(): string {
-    return this._translateService.currentLang;
+    return aggregationFilters === undefined ? [] : aggregationFilters.values;
```

Comment:
COSMETIC : ternary instruction on 3 lines seems more readable at my opinion.
I was my changes in very old PR (12/2020)